src/screens/Main.tsx

```diff
@@ -6,27 +6,41 @@ import Catalog from "../components/Catalog";
 import PokeCard from "../components/PokeCard";
 
 const Main = () => {
-  const [pokedex, setPokedex] = useState([]);
+
+  type Pokedex = {
```

Comment:
Hi Kurt, let's transfer these to a separate file. Let's do `types.ts` and just export them from there.